static/css/style.css

```diff
@@ -121,7 +121,8 @@ h2 {
   border-radius: 0.5rem;
 }
 
-#small-icons > *:not(:first-child) {
+#small-icons-left > *:not(:first-child),
+#small-icons-right > *:not(:first-child) {
```

Comment:
Since we've created the small icons class we can replace this with `.small-icons > *:not(:first-child)`.

Reply:
The same would apply to all the other `#small-icons-left/right` below.

Reply:
If I change all the IDs to `.small-icons` i get overlapping buttons<img width="272" alt="Screenshot 2019-11-20 at 15 00 13" src="https://user-images.githubusercontent.com/31242877/69249848-8316af00-0ba6-11ea-803f-de088d071dcf.png">

removing  `.small-icons {width: 54px;}` appears to fix this, is that okay

Reply:
They should be exactly the same, could you update the PR with just the name change? Maybe there is something left in the codebase we missed.

Reply:
I've updated the PR with the changes

Reply:
The class priority order changed and the padding applied to the `a` elements was trumping the `.small-icons` class, so I added a couple of `!importants`. Not a perfect solution, but it will do